src/main/resources/db/changelog/changes/v1.2.0/changelog-v1.2.0.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+  <changeSet id="MODLISTS-123" author="mweaver@ebsco.com" runOnChange="true">
+    <comment>Update the canned lists to use the new entity type IDs and column names</comment>
+    <sql>
+      INSERT INTO list_details
+      (id, name, description, entity_type_id, fql_query, fields, created_by, created_by_username, created_date, updated_by, updated_by_username, updated_date, is_active, is_private, is_canned, version, user_friendly_query)
+      values
+        ('605a345f-f456-4ab2-8968-22f49cf1fbb6',
+         'Missing items',
+         'Returns all items with a status of: missing, aged to lost, claimed returned, declared lost, long missing',
+         'd0213d22-32cf-490f-9196-d81c3c66e53f',
+         '{"items.status_name": {"$in": ["Missing", "Aged to lost", "Claimed returned", "Declared lost", "Long missing" ] }}',
+         '{id, item_hrid, holdings_id, item_effective_call_number, item_effective_call_number_typeid, item_effective_call_number_type_name, item_holdings_record_id, item_status, item_copy_number, item_barcode, item_created_date, item_updated_date, item_effective_location_id, item_effective_location_name, item_effective_library_id, item_effective_library_name, item_effective_library_code, item_material_type_id, item_material_type, instance_id, instance_title, instance_created_date, instance_updated_date, instance_primary_contributor, item_level_call_number, item_level_call_number_typeid, item_permanent_location_id, item_temporary_location_id, item_level_call_number_type_name, item_permanent_location_name, item_temporary_location_name}',
+         '7eb41f7c-ec2f-4637-8ceb-2e9573666ad0',
+         'SYSTEM',
+         Now(),
+         '7eb41f7c-ec2f-4637-8ceb-2e9573666ad0',
+         'SYSTEM',
+         Now(),
+         true,
+         false,
+         true,
+         1,
+         'items.status_name in [Missing, Aged to lost, Claimed returned, Declared lost, Long missing]'),
+
+        ('97f5829f-1510-47bc-8454-ae7fa849baef',
+         'Inactive patrons with open loans',
+         'Returns all inactive users with a loan status of open',
```

Comment:
I stealthily changed the list name and description, since they were wrong before (it was "Inactive patrons with open loans" for the title and something similar for the description)